.pre-commit-config.yaml

```diff
@@ -27,12 +27,17 @@ repos:
         files: ^api/src/.*\.py$
 
   - repo: https://github.com/PyCQA/bandit
-    rev: 1.6.2
+    rev: 1.7.4
     hooks:
       - id: bandit
         args: [-l, --recursive, -x, tests]
         files: ^api/.*\.py$
 
+  - repo: https://github.com/Lucas-C/pre-commit-hooks-safety
+    rev: v1.2.4
```

Comment:
Some risk trusting this relative small repo.
Since we already use .venv in another hook, we can do that here as well.
OR, up to you, trust this guy, but increase security by pinning to a commit sha.

Reply:
I agree and I thought about it in the back of my mind. Changing it to run in .venv.